pkgs/misc/vim-plugins/update.py

```diff
@@ -407,6 +462,12 @@ def parse_args():
         default=DEFAULT_OUT,
         help="Filename to save generated nix code",
     )
+    parser.add_argument(
```

Comment:
If we suggest to do this anyway, why not default to `true`?

Reply:
I don't know exactly what the use case for the `--input-file` argument of this script is so I was cautious. It seems designed so one could maintain their own plugin list and it would be unexpected for this script to magically rewrite their file.

Reply:
I agree. We don't need an option here. Just make it the default.
If someone complains about the new behavior with a good reason, we can still add it back.

Reply:
Your point is valid, but this is a nixpkgs-internal tool and I don't think we need to prioritize backwards compatibility. If someone else is using this outside of nixpkgs, its their responsibility to keep up with upstream changes. If we wanted to advertise this outside of nixpkgs we'd make it into its own package (with semver).

Reply:
The flag is removed.